backend/controllers/BroadcastController.ts

```diff
@@ -76,11 +77,50 @@ async function updateTwilioStatus(req: Request, res: Response) {
   return AppResponse.ok(res, {}, 204)
 }
 
+async function commentChangeSubscription(req: Request, res: Response) {
```

Comment:
It should have a better name to indicate you're only targeting `Unsubscribe` or `Resubscribe`